web/src/routes/stall/[...slug].svelte

```diff
@@ -42,18 +38,30 @@
 </script>
 
 
-{#if $user && $user.nym === stallName || stallName === ""}
-    <ListView
+{#if $user && $user.nym === stallOwnerNym || stallOwnerNym === ""}
+    <!-- User viewing own store -->
+    <StallView
         title="My Stall"
-        loader={{endpoint: 'auctions', fromJson}} newEntity={() => new Auction()}
-        card={AuctionCard} editor={AuctionEditor} bind:showNewButton={showNewButton}
+        loader={{endpoint: 'auctions', responseField: 'auctions', fromJson}}
```

Comment:
Any reason to use `'auctions'` endpoint rather than the new `/users/<nym>/auctions`? I think the two endpoints are now trying to do the same thing, so we should get rid of the old one. Of course this would mean we need to pass `for_user=<the user making the request>` to to_dict()

Reply:
well the reason was if there was no nym - but i am fixing that elsewhere so i dont think so